frontend_app_agenda/src/css/index.styl

```diff
@@ -9,3 +9,15 @@
   height 100%
 .pageContentGeneric.agendaPage
   height calc(100% - 135px)
+
+.agendaPage
+  &__warningMessage
+    position absolute
+    right 25px
+    top 110px
+
+// MEDIA QUERIES
```

Comment:
You can safely leave out this comment I think :-)

Reply:
I put because almost all components have, but I agree with you

Reply:
Ah, ok :-)